src/mgr/BaseMgrModule.cc

```diff
@@ -565,13 +565,19 @@ ceph_cluster_log(BaseMgrModule *self, PyObject *args)
 }
 
 static PyObject *
-ceph_get_version(BaseMgrModule *self, PyObject *args)
```

Comment:
nit, this change is not correct. we should keep `PyObject *args`, even it's not used.

quote from https://docs.python.org/3/c-api/structures.html#c.PyCFunction

> Type of the functions used to implement most Python callables in C. Functions of this type take two PyObject* parameters and return one such value.